unit_tests/README.md

```diff
@@ -0,0 +1,16 @@
+# Unit Testing Netguard
+
+## Requirements
+An Android x86/64 emulator with no Google Play
+
+## Running Tests
+* Build the app or just the `vpn-network-imp` module
```

Comment:
The `test_tcp` is not compiled.
I have tried `./gradlew :vpn-network-impl:assemble`.
The `libnetguard.so` library is in there, not the test executable